src/utils.cxx

```diff
@@ -508,6 +508,20 @@ int get_padding_length(const std::string& name) {
     return count;
 }
 
+std::string strip_padding(const std::string &lib)
+{
+    // One of the padding characters is a legitimate
+    // path separator
+    int pad_len = get_padding_length(lib)-1;
```

Comment:
I think we talked about it before, but this method and `get_padding_length` presume the path is always of the form `<driveletter>:/...` (vs. a network drive)

I think that should be documented - here or in pad_path (and if in pad_path, this should point to that function and say something like "assumes path is of the form described in pad_path").

Also:

> // One of the padding characters is a legitimate
> // path separator

I think that should be accounted for in `get_padding_length` (so that callers don't always have to `-1` the result).